setup.py

```diff
@@ -77,6 +75,7 @@ def read_version():
         "fabric>=2.0",
         "requests>=2.20.0, <3",
         "sagemaker-experiments",
+        "pandas",
```

Comment:
actually this is already in the extra["test"] because this line. we need to add this to  required_packages
```
extras["test"] = (
    [
        extras["all"],
```

Reply:
so just remove from the special use case and add to required?